src/nvim/ex_cmds2.c

```diff
@@ -744,40 +744,42 @@ void dbg_breakpoint(char_u *name, linenr_T lnum)
 
 static char_u   *profile_fname = NULL;
 
-/*
- * ":profile cmd args"
- */
+/// ":profile cmd args"
+///
+/// @param eap Pointer to the command arguments structure.
 void ex_profile(exarg_T *eap)
 {
-  static proftime_T pause_time;
+  static proftime_T  pause_time;
+  int                len;
+  char_u            *arg;
```

Comment:
It's not even C89 (:-P), but I get the point.

The `char_u` -> `char`/`uint8_t` change will make lots of code much cleaner. That will be a joyful day!